tests/common/test_logger.py

```diff
@@ -621,11 +621,14 @@ def test_file_appender_with_logrotate_not_supported_archiving_backup(self):
             log_file_path = os.path.join(self.log_folder, log_file)
 
             # If the files are not gzip, the gzip library throws an OSError: Not a gzipped file.
-            with gzip.open(log_file_path, 'rb') as logfile:
-                logcontent = map(lambda x: x.strip(), logfile.readlines())
-                for logline in logcontent:
-                    # Only 7-13 are in the logs. Others shouldn't be in the logs.
-                    self.assertRegex(logline.decode('utf-8'), r".*INFO\s\w+\s*test-info-1?[0-9]")
+            logfile = gzip.open(log_file_path, 'rb')
```

Comment:
Same comment: https://stackoverflow.com/questions/6086603/statement-with-and-tarfile/6086722#6086722

Reply:
Thanks for this.